packages/core/src/Entity.ts

```diff
@@ -433,11 +436,13 @@ export class Entity extends EngineObject {
     this._templateResource = templateResource;
   }
 
-  private _createCloneEntity(srcEntity: Entity): Entity {
+  private _createCloneEntity(): Entity {
```

Comment:
Is this modification necessary? or is this related to prefab?
I think you should minimize your modification.